cucumber/features/steps/devLoginPupSteps.js

```diff
@@ -0,0 +1,44 @@
+const {
```

Comment:
This test file doesn't use the selenium webdriver, should it?

Reply:
No, the webdriver was used in the first attempt. The idea was to remove the files with it. I'll remove them.